### PR TITLE
Update extensions.md

### PR DESCRIPTION
**Pull request by joseph619**
Added new FAQ entry and minor fixes for Manga Rock.

### CODE REVIEW

File: src/help/faq/extensions.md [hunk 1]
```diff
@@ -182,14 +182,22 @@ Read about source migration [here](/help/guides/source-migration/)
...
+Read more about the situation [here](https://mangarock.com/).
 :::
 
 #### Migrate to Tachiyomi
 There is a tool to migrate from **MangaRock** to **Tachiyomi** courtesy of a member of our [Discord server](https://discord.gg/tachiyomi) by the same name as the creator, but a computer is required to finish the migration successfully.
 
 View the program at [waicool20/mr2tachiyomi](https://github.com/waicool20/mr2tachiyomi).
 
+#### Restore backup from Manga Rock source
+It's currently not possible to restore backup from Manga Rock officially, as Manga Rock source is dead. There's an unofficial method to restore backup all manga from Manga Rock source.
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
It's not currently possible to restore backup from MangaRock officially, as the MangaRock source is dead. There is howeven an unofficial method to restore/backup all manga from the MangaRock source.
```

File: src/help/faq/extensions.md [hunk 2]
```diff
@@ -182,14 +182,22 @@ Read about source migration [here](/help/guides/source-migration/)
...
+
+1. Uninstall Manga Rock extension.
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
1. Uninstall the MangaRock extension.
```

File: src/help/faq/extensions.md [hunk 3]
```diff
@@ -182,14 +182,22 @@ Read about source migration [here](/help/guides/source-migration/)
...
+1. Install Dummy Rock apk. Link: [Dummy Rock](https://cdn.discordapp.com/attachments/403520500443119619/663671637052489728/tachiyomi-en.dummyrock-v1.2.1-debug.apk)
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
1. Install the [DummyRock](https://cdn.discordapp.com/attachments/403520500443119619/663671637052489728/tachiyomi-en.dummyrock-v1.2.1-debug.apk) `.apk`. 
```

File: src/help/faq/extensions.md [hunk 4]
```diff
@@ -182,14 +182,22 @@ Read about source migration [here](/help/guides/source-migration/)
...
+1. Go to **[<MaterialIcon icon-name="extension"/> Extensions](/help/guides/extensions)** menu and press **Trust** to trust Dummy Rock.
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
1. Go to **[<MaterialIcon icon-name="extension"/> Extensions](/help/guides/extensions)** menu and press **Trust** to proceed.
```

File: src/help/faq/extensions.md [hunk 5]
```diff
@@ -182,14 +182,22 @@ Read about source migration [here](/help/guides/source-migration/)
...
-Read more about the situation [here](https://mrcomics.com/).
+Read more about the situation [here](https://mangarock.com/).
 :::
 
 #### Migrate to Tachiyomi
 There is a tool to migrate from **MangaRock** to **Tachiyomi** courtesy of a member of our [Discord server](https://discord.gg/tachiyomi) by the same name as the creator, but a computer is required to finish the migration successfully.
 
 View the program at [waicool20/mr2tachiyomi](https://github.com/waicool20/mr2tachiyomi).
 
+#### Restore backup from Manga Rock source
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
#### Restore backup from the MangaRock source
```

File: src/help/faq/extensions.md [hunk 6]
```diff
@@ -182,14 +182,22 @@ Read about source migration [here](/help/guides/source-migration/)
...
+It's currently not possible to restore backup from Manga Rock officially, as Manga Rock source is dead. There's an unofficial method to restore backup all manga from Manga Rock source.
+
+1. Uninstall Manga Rock extension.
+1. Install Dummy Rock apk. Link: [Dummy Rock](https://cdn.discordapp.com/attachments/403520500443119619/663671637052489728/tachiyomi-en.dummyrock-v1.2.1-debug.apk)
+1. Go to **[<MaterialIcon icon-name="extension"/> Extensions](/help/guides/extensions)** menu and press **Trust** to trust Dummy Rock.
+1. Restore Backup
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
1. Restore backup.
```

File: src/help/faq/extensions.md (hunk 1 above)
Comment (joseph619) [reply src/help/faq/extensions.md]: All changes made

File: src/help/faq/extensions.md [hunk 7]
```diff
@@ -182,14 +182,22 @@ Read about source migration [here](/help/guides/source-migration/)
...
-Read more about the situation [here](https://mrcomics.com/).
+Read more about the situation [here](https://mangarock.com/).
 :::
 
 #### Migrate to Tachiyomi
 There is a tool to migrate from **MangaRock** to **Tachiyomi** courtesy of a member of our [Discord server](https://discord.gg/tachiyomi) by the same name as the creator, but a computer is required to finish the migration successfully.
 
 View the program at [waicool20/mr2tachiyomi](https://github.com/waicool20/mr2tachiyomi).
 
+#### Restore backup from  the MangaRock source
```
Comment (Soitora) [inline src/help/faq/extensions.md]:
```suggestion
#### Restore a backup from the MangaRock source
```

Comment (joseph619) [reply src/help/faq/extensions.md]: Done

**Pull request merged**